solidity/test/integration/ConnextVestingWallet.t.sol

```diff
@@ -7,16 +7,23 @@ import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
 import {ConnextVestingWallet} from 'contracts/ConnextVestingWallet.sol';
 import {IntegrationBase} from 'test/integration/IntegrationBase.sol';
 
+/**
+ * TODO: Make this tests more generic and clear
+ */
 contract IntegrationConnextVestingWallet is IntegrationBase {
```

Comment:
blz remove TODO comment, this test has no need of inheriting `Deploy`, we shouldn't name it an "integration" test, let's consider this a more "unit" test on the implementation. Blz let's keep a separations of affairs, this one just deploys the contract, sends tokens, and checks the 1/13s unlock schedule, and the llama test actually deploys the Script, and does precise expectations to the amounts 🙏